robottelo/host_helpers/cli_factory.py

```diff
@@ -790,14 +790,33 @@ def _setup_org_for_a_rh_repo(self, options=None):
         except CLIReturnCodeError as err:
             raise CLIFactoryError(f'Failed to fetch content view info\n{err.msg}') from err
         # Promote version1 to next env
-        try:
-            self._satellite.cli.ContentView.version_promote(
-                {'id': cvv['id'], 'organization-id': org_id, 'to-lifecycle-environment-id': env_id}
-            )
-        except CLIReturnCodeError as err:
-            raise CLIFactoryError(
-                f'Failed to promote version to next environment\n{err.msg}'
-            ) from err
+        if force:
+            try:
+                self._satellite.cli.ContentView.version_promote(
+                    {
+                        'id': cvv['id'],
+                        'organization-id': org_id,
+                        'to-lifecycle-environment-id': env_id,
+                    }
+                )
+            except CLIReturnCodeError as err:
+                raise CLIFactoryError(
+                    f'Failed to promote version to next environment\n{err.msg}'
+                ) from err
+        else:
+            try:
+                self._satellite.cli.ContentView.version_promote(
+                    {
+                        'id': cvv['id'],
+                        'organization-id': org_id,
+                        'to-lifecycle-environment-id': env_id,
+                        'force': True,
+                    }
+                )
+            except CLIReturnCodeError as err:
+                raise CLIFactoryError(
+                    f'Failed to promote version to next environment\n{err.msg}'
+                ) from err
```

Comment:
You should not need `if .. else` here, just pass the force parameter as it is be its `False` or `True`